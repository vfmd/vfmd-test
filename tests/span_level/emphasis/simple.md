Using asterisks: *em* and **strong** emphasis

Using underscores: _em_ and __strong__ emphasis
