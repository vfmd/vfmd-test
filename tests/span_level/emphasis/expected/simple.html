<p>Using asterisks: <em>em</em> and <strong>strong</strong> emphasis</p>

<p>Using underscores: <em>em</em> and <strong>strong</strong> emphasis</p>
